src/components/inputs/input/Input.stories.tsx

```diff
@@ -0,0 +1,134 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { JSX, useState } from 'react';
+
+import { Meta, StoryObj } from '@storybook/react';
+
+import { Input } from './Input';
+import { Icon } from '../../basic/icon/Icon';
+import { Text } from '../../basic/text/Text';
+import { Container } from '../../layout/Container';
+
+const meta = {
+	component: Input,
+	args: {
+		label: 'Input label'
+	}
+} satisfies Meta<typeof Input>;
+export default meta;
+
+type Story = StoryObj<typeof Input>;
+
+export const Default = {} satisfies Story;
```

Comment:
This default inherit the label arg from the meta. I would prefer to have a Default story with only the required props (none for the input)